map.html

```diff
@@ -62,48 +60,42 @@
 <div id="map"></div>
 <script>
     function initMap() {
-        map = new google.maps.Map(document.getElementById('map'), {
+        // initialize Leaflet
+        var mainLayer = L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+            maxZoom: 19, attribution: '&copy; <a href="https://openstreetmap.org/copyright">OpenStreetMap contributors</a>'
+        });
+
+        map = L.map('map', {
+            center: {lat: 0, lon: 0},
             zoom: 3,
-            center: {lat: 0, lng: 0}
+            layers: [mainLayer]
         });
 
-        function addPin(i, chatizen) {
-            var marker = new google.maps.Marker({
-                position: {lat: chatizen.lat, lng: chatizen.lon},
-                map: map,
-                label: chatizen.nick[0],
-                title: chatizen.nick
-            });
+        // show the scale bar on the lower left corner
+        L.control.scale().addTo(map);
 
-            var infowindow = new google.maps.InfoWindow({
-                content: chatizen.nick
-            });
-
-            marker.addListener('click', function () {
-                infowindow.open(map, marker);
-            });
+        function addPin(chatizen) {
+            L.marker({lat: chatizen.lat, lon: chatizen.lon}, {riseOnHover: true})
+                .bindPopup(chatizen.nick, {autoClose: false, closeOnClick: false})
+                .bindTooltip(chatizen.nick, {/*permanent: true*/}).openTooltip()
+                .addTo(map);
         }
 
         document.cookie = "password=sadmcasldkfjsdclasdkcmascdmklasdm";
-        $.getJSON(
-            "https://chatmap.dbco.link/api/chatizen",
-            function (data, status, xhr) {
-                var options = {
-                    keys: ["nick"],
-                    id: "nick"
-                };
-
-                chatizens = data;
+        fetch("https://chatmap.dbco.link/api/chatizen").then(response => response.json()).then((data) => {
```

Comment:
The call to fetch() needs a credentials: "include" argument to work

Reply:
Should also change it to `fetch("/api/chatizen", {credentials: "include"})` while we're at it, to make it a relative URL. This simplifies testing.